### ISSUE TITLE
Laravel 5

### ISSUE THREAD

**Issue by anlutro** [anlutro/laravel-4-smart-errors#24]
Laravel 5 has stripped out the error handling features that made this package work out of the box. Options:

- Wait for features to be re-implemented (not likely to happen)
- Re-implement it ourselves (too much hassle)
- Add a simple wrapper method that sort of replicates the old behaviour, making it possible to call `SmartErrors::handle($exception)` in the user's exception handler (loses many features)

Additionally, config namespacing has been removed, so that will need to be reworked.

**Comment by Bartjuh4** [comment]
Maybe this package can help:

https://github.com/morrislaptop/LaravelFivePackageBridges

This weekend i will give it a try and share my results.

**Comment by anlutro** [comment]
Thanks for your initial work on this - it's very useful.

I won't follow the same approach - I don't want to maintain two different versions of the library for different Laravel versions, so I'll have to add some extra code to provide compatibility between the versions.

**Comment by Bartjuh4** [comment]
Okay, maybe i could help you with that. The goal to get this package working for L4 as wel as L5 would be great. The only change (in my opinion) that may be needed at this moment would be a switch in the serviceprovider for L4/L5.

**Comment by anlutro** [comment]
Indeed. The initial work I did is on the develop branch if you're interested.

I just need to decide how to do the ErrorHandler class really. The `report()` and `render()` stuff really doesn't fit with my model of how errors should be managed, so I need to think about it.
